lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,6 +55,10 @@
   --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
   --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
 
+  //NxTooltip variables
+  --nx-color-tooltip-background: var(--nx-swatch-grey-05);
```

Comment:
Suppose this needs to be `grey-20` as well?

Reply:
> Suppose this needs to be `grey-20` as well?

Oops, I'd made the change locally and forgot to push! [9eb9c0c](https://github.com/sonatype/sonatype-react-shared-components/pull/974/commits/9eb9c0c04f607c4b3ab51325c151ad453eae1520)